zone_example_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"log"
 
+	"golang.org/x/net/context"
```

Comment:
Different context module?

Reply:
looks to be similar (if not the same thing).

- https://pkg.go.dev/context
- https://pkg.go.dev/golang.org/x/net/context

different domain perhaps?

i'll remove it either way as stdlib covers this.

Reply:
9b62079

Reply:
ah, the second is deprecated since 1.7 as it's now in stdlib. i wonder why this got pulled in 🤔